docs/rules/arrow-parens.md

```diff
@@ -176,4 +179,45 @@ a.then(foo => { if (true) {}; });
 (a = 10) => a;
 ([a, b]) => a;
 ({a, b}) => a;
-```
\ No newline at end of file
+```
+
+
+### requireForBlockBody
+
+When the rule is set to `"as-needed", { "requireForBlockBody": true }` the following patterns are considered problems:
```

Comment:
To help our documentation generator, can you change this line to read

> Examples of **incorrect** code for the `{ "requireForBlockBody": true }` option:

The line introducing the correct examples below should be similar, saying "correct" instead of "incorrect".